middlewares/index.js

```diff
@@ -62,11 +62,6 @@ module.exports = {
 
           // if everything is good, save to request for use in other routes
           req.decoded = decoded; 
-
-          // so that entity can perform database operations (RUD) 
-          // only on his data
-          req.body.email = decoded.email;
```

Comment:
this change?

Reply:
so this will override any value in body e.g. this was adding `id` in `service` object and throwing error in sequelize. We shouldn't change request body instead use req.decoded whenever necessary in routes.